cmd/cosign/cli/sign/sign_blob.go

```diff
@@ -82,6 +98,20 @@ func SignBlobCmd(ro *options.RootOptions, ko options.KeyOpts, regOpts options.Re
 		signedPayload.Bundle = cbundle.EntryToBundle(entry)
 	}
 
+	// if bundle is specified, just do that and ignore the rest
+	if ko.RFC3161TimestampPath != "" {
```

Comment:
Should we require that if `ko.TSAServerURL` is set, `ko.RFC3161TimestampPath` must also be? Otherwise it's unnecessarily fetching a timestamp without persisting it.